Automattic/jetpack
# Dashboard: fix some styles in AAG cards.

## Proposed changes:

Following the redesign of the Jetpack dashboard, this little style adjustment aims to fix ~2~ 1 layout issue in the cards appearing below the stats graph in the At a Glance dashboard:

**Scan card**

| Before | After |
| ------ | ----- |
| <img width="632" alt="Screenshot 2023-04-27 at 15 02 49" src="https://user-images.githubusercontent.com/426388/234877547-353cbd88-d2b5-426a-b42a-299612251455.png"> | <img width="599" alt="Screenshot 2023-04-27 at 15 02 54" src="https://user-images.githubusercontent.com/426388/234877578-2734ca84-a50d-4a67-b776-eed835a7834e.png"> |

### Other information:


## Jetpack product discussion

* p1HpG7-lx8-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* You'll want to start with a site running this branch, connected to WordPress.com, and where you have purchased a Jetpack Scan Daily plan.
* Go to Jetpack > Dashboard
* Check the cards below the stats graph, following the screenshots above.
